consul/audit/index.js

```diff
@@ -0,0 +1,13 @@
+var express = require('express')
+var app = express()
+
+app.get('/', function (req, res) {
+    res.send('Hello World!')
+})
+app.get('/health', function (req, res) {
+    res.send('I am healthy!')
```

Comment:
@esbie didn't notice the helloworld app, i've removed the node app and use that instead. Might be useful to add a contributors.md with these suggestions.

Consul can/should use health checks, but actually my example doesn't configure them. In a future update i'll add consul health checks and update the go hello world to support a health check endpoint